### PR TITLE
Fixes uniforms + captain's chair

### LINKED ISSUE #966: Shuttle recall is borked
AHHHHHh

### PR DESCRIPTION
**Pull request by Vivalas**
Couldn't fix shuttle yet but at least got these done

Fixes #966

### CODE REVIEW

File: code/modules/clothing/under/jobs/security.dm [hunk 1]
```diff
@@ -14,38 +14,37 @@
...
 	strip_delay = 50
 	alt_covers_chest = 1
 	sensor_mode = 3
 	random_sensor = 0
 
-
+
 /obj/item/clothing/under/rank/masteratarms/grey
-	icon_state = "masteratarms"
+	icon_state = "warden"
```
Comment (ike709) [inline code/modules/clothing/under/jobs/security.dm]: u wot m8

File: code/modules/jobs/job_types/assistant.dm [hunk 2]
```diff
@@ -31,14 +31,13 @@ Assistant
...
+	jobtype = /datum/job/assistant
 
 /datum/outfit/job/assistant/pre_equip(mob/living/carbon/human/H)
 	..()
-	if (config.grey_assistants)
-		uniform = /obj/item/clothing/under/color/grey
 
-	else
-		uniform = /obj/item/clothing/under/color/random
+	uniform = /obj/item/clothing/under/color/grey //gotta get those uniforms
```
Comment (ike709) [inline code/modules/jobs/job_types/assistant.dm]: You removed rainbowtide?

File: code/modules/clothing/under/jobs/security.dm (hunk 1 above)
Comment (Vivalas) [inline code/modules/clothing/under/jobs/security.dm]: Forgot about this

**Comment by Vivalas** [comment]
Just fixed recall too

**Comment by ike709** [comment]
add Fixes #966 to the OP

**Pull request merged**